docs/collector-limitations.md

```diff
@@ -0,0 +1,8 @@
+# Elastic OpenTelemetry Collector limitations

```

Comment:
For titles:
Elastic Distribution for OpenTelemetry Collector or 
Elastric Distro for OpenTelemetry Collector or

For body when we really need to shorten it
Elastic OTel Collector

Reply:
@ishleenk17 can you please review this section?

Reply:
>For titles:
Elastic Distribution for OpenTelemetry Collector or
Elastric Distro for OpenTelemetry Collector or

>For body when we really need to shorten it
Elastic OTel Collector

As proposed in the slack channel:
| Full name                                            | Shortened name for docs body text  |
|------------------------------------------------------|------------------------------------|
| Elastic Distributions for OpenTelemetry              | EDOT                               |
| Elastic Distribution for OpenTelemetry Collector     | EDOT Collector                     |
| Elastic Distribution for OpenTelemetry Java Agent   | EDOT Java Agent                    |
| Elastic Distribution for OpenTelemetry .NET SDK        | EDOT .NET SDK                      |